requirements-base.txt

```diff
@@ -3,3 +3,4 @@ oauthlib>=1.0.3
 requests-oauthlib>=0.6.1
 six>=1.10.0
 PyJWT>=1.4.0
+cryptography>=1.4
```

Comment:
Maybe there's something I don't understand about what this package exports - but how is it used in this PR / why was it added?

Reply:
This is required by PyJWT to use RSA encryption. https://pyjwt.readthedocs.io/en/latest/installation.html#cryptographic-dependencies-optional